GetQueueSpidMessages/function.json

```
@@ -0,0 +1,13 @@
+{
+  "bindings": [
+    {
+      "queueName": "spid-msg-items",
+      "connection": "QueueStorageConnection",
+      "name": "spidMsgItem",
+      "type": "queueTrigger",
+      "direction": "in"
+    }
+  ],
+  "disabled": false,
+  "scriptFile": "../dist/GetQueueSpidMessages/index.js"
```

the real action here is "store spid messages" so I'd rather use something like 

```suggestion
  "scriptFile": "../dist/StoreSpidLogs/index.js"
```